src/platform/windows/window.rs

```diff
@@ -14,6 +14,7 @@ use winapi::um::{combaseapi, dwmapi, libloaderapi, winuser};
 use winapi::um::objbase::COINIT_MULTITHREADED;
 use winapi::um::shobjidl_core::{CLSID_TaskbarList, ITaskbarList2};
 use winapi::um::winnt::{LONG, LPCWSTR};
+use winapi::um::wingdi::CreateRectRgn;
```

Comment:
Since `wingdi` is alphabetically before `winnt`, this import should be before it.